Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/SubscribeTransforms.java

```diff
@@ -62,7 +62,6 @@ public LogEventRequestType transformNhinSubscribeRequestToAuditMessage(LogNhinSu
         AuditMessageType auditMsg = new AuditMessageType();
```

Comment:
This class can be removed. These are part of HIEM and that service implementation is removed from codebase.